data/monster/monsters/ghulosh'_deathgaze.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<monster name="Ghulosh' Deathgaze" nameDescription="Ghulosh' Deathgaze" race="undead" experience="0" speed="30">
```

Comment:
is speed correct? 30 is really really slow

Reply:
yes